apps/token/src/routes/staking/node/node.tsx

```diff
@@ -137,4 +128,8 @@ export const StakingNode = ({ data }: StakingNodeProps) => {
   );
 };
 
-export default StakingNodeContainer;
+export const Node = () => {
+  return (
+    <NodeContainer>{({ data }) => <StakingNode data={data} />}</NodeContainer>
```

Comment:
Question (blocking?): Intentional closing brace as the only content in the NodeContainer?